ui/perfherder/helpers.js

```diff
@@ -702,27 +697,29 @@ export const getAlertSummaryStatusText = alertSummary =>
     status => status.id === alertSummary.status,
   ).text;
 
-export const saveNotes = alertSummary =>
-  modifyAlertSummary(alertSummary, { notes: alertSummary.notes }).then(() => {
-    alertSummary.originalNotes = alertSummary.notes;
-    alertSummary.notesChanged = false;
-  });
+export const saveNotes = async alertSummary => {
+  await modifyAlertSummary(alertSummary, { notes: alertSummary.notes });
+  alertSummary.originalNotes = alertSummary.notes;
+  alertSummary.notesChanged = false;
+};
 
 export const editingNotes = alertSummary => {
   alertSummary.notesChanged = alertSummary.notes !== alertSummary.originalNotes;
 };
 
-export const getAlertSummary = id =>
-  OptionCollectionModel.getMap().then(optionCollectionMap =>
-    getData(getApiUrl(`/performance/alertsummary/${id}/`)).then(({ data }) =>
-      AlertSummary(data, optionCollectionMap),
-    ),
-  );
+export const getAlertSummary = async id => {
+  const optionCollectionMap = await OptionCollectionModel.getMap();
+  const { data } = await getData(getApiUrl(`/performance/alertsummary/${id}/`));
 
-export const getAlertSummaryTitle = id =>
-  getAlertSummary(id).then(alertSummary => getTitle(alertSummary));
+  return AlertSummary(data, optionCollectionMap);
+};
 
-export const getAlertSummaries = options => {
+export const getAlertSummaryTitle = async id => {
+  const alertSummary = await getAlertSummary(id);
+  return getTitle(alertSummary);
+};
```

Comment:
I'm not sure there's much benefit in switching this small function to async/await. I think its more useful when there are multiple nested .then()'s :)

Reply:
+1